webapp/src/selectors.ts

```diff
@@ -125,6 +125,15 @@ export const voiceChannelCallHostChangeAt = (state: GlobalState, channelID: stri
     return pluginState(state).voiceChannelCalls[channelID]?.hostChangeAt;
 };
 
+export const voiceChannelCallDismissedNotification = (state: GlobalState, channelID: string) => {
+    const dismissed = pluginState(state).voiceChannelCalls[channelID]?.dismissedNotification;
+    if (!dismissed) {
+        return false;
+    }
+    const currentUserID = getCurrentUserId(state);
+    return (dismissed.indexOf(currentUserID) !== -1);
```

Comment:
It's 2023, could we use `includes()` :p ?

Reply:
You're no fun. 😝